brainstormforce/ultimate-addons-for-gutenberg
# Fix: CF7-Styler - Submit button text color issue.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/325badbd-844e-4536-8ac8-79c15c7c3730?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop CF7 styler and add a text color to submit button in next-release to reproduce the issue.
- Switch to the above branch and refresh the frontend and editor. Now you will see text color applying to the submit button text.

### Checklist:
